### PR TITLE
Give all Miro works a work type of "Digital Image"

### LINKED ISSUE #2477: Give all Miro works a work type of "Digital image"

### PR DESCRIPTION
**Pull request by alexwlchan**
Resolves #2477.

### CODE REVIEW

File: catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/miro/MiroWorkType.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  /** We set the same work type on all Miro images.
+    *
+    * This is based on the Sierra work types -- we'll want to revisit this
+    * when we sort out work types properly, but it'll do for now.
+    */
+  def getWorkType: Option[WorkType] =
+    Some(WorkType(
+      id = "q",
+      label = "Digital Images"
```
Comment (jtweed) [inline catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/miro/MiroWorkType.scala]: For consistency, this comes through from Sierra as `Digital images`. Obviously all this changes when we're picking up from our own list, like location types.

**Comment by alexwlchan** [comment]
Ready to merge.

File: catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/transformers/MiroTransformableTransformerTest.scala [hunk 2]
```diff
@@ -273,6 +273,12 @@ class MiroTransformableTransformerTest
     work.items.head.agent.locations shouldBe List(expectedDigitalLocation)
   }
 
+  it("sets the WorkType as 'Digital images'") {
+    val work = transformWork()
+    work.workType.isDefined shouldBe true
+    work.workType.get.label shouldBe "Digital images"
```
Comment (mklander) [inline catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/transformers/MiroTransformableTransformerTest.scala]: maybe
`work.workType shouldBe defined`
or
`work.workType shouldBe Some(WorkType("q", "Digital images"))`?

not sure if the `id` matters

Comment (alexwlchan) [reply catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/transformers/MiroTransformableTransformerTest.scala]: It's sort of arbitrary that the ID is "q" – it's cribbed from the Sierra work types spreadsheet.

I didn't want to write something that was a literal clone of the code under test.

I don't think it makes much of a difference.

Comment (mklander) [reply catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/transformers/MiroTransformableTransformerTest.scala]: Sure, if it doesn't matter don't test for it.

**Pull request merged**